openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +485,67 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is handled by the Request URI endpoint of the Verifier.  
+
+The request MUST use the HTTP POST method with the https scheme, and the content type `application/x-www-form-urlencoded` and the accept header set to `application/oauth-authz-req+jwt`.
```

Comment:
```suggestion
The request MUST use the HTTP POST method with the `https` scheme, and the content type `application/x-www-form-urlencoded` and the accept header set to `application/oauth-authz-req+jwt`.
```

I am sorry, I think I missed a discussion on the accept header, but JAR does not seem to mandate `application/oauth-authz-req+jwt`:

> A [Request Object](https://datatracker.ietf.org/doc/html/rfc9101#request_object) ([Section 2.1](https://datatracker.ietf.org/doc/html/rfc9101#request_object)) has the media type [[RFC2046](https://datatracker.ietf.org/doc/html/rfc2046)] application/oauth-authz-req+jwt. Note that some existing deployments may alternatively be using the type application/jwt.

do we really want to mandate it..?